ckan/Dockerfile

```diff
@@ -31,11 +24,18 @@ ENV PIP_SRC=${SRC_DIR}
 ENV CKAN_STORAGE_PATH=/var/lib/ckan
 RUN mkdir -p $CKAN_STORAGE_PATH $APP_DIR
 
+ADD Pipfile Pipfile.lock ${APP_DIR}/
+ENV PIPENV_PIPFILE=/${APP_DIR}/Pipfile
+RUN pip3 install --upgrade pipenv && pipenv install --ignore-pipfile
```

Comment:
Why are we ignoring pipfile here? I think we want to use that...

Reply:
We're ignoring `pipfile` in favor of `pipfile.lock` which is the hard set of requirements.